shared-helpers/src/locales/es.json

```diff
@@ -750,7 +750,7 @@
   "listings.applications": "Solicitudes",
   "listings.apply.applicationWillBeAvailableOn": "La solicitud podrá ser descargada y recogida el %{openDate}",
   "listings.apply.applyOnline": "Haga su solicitud por Internet",
-  "listings.apply.applyOnlineMessage": "<to add after Localize translation>",
```

Comment:
listado? Fine with me if we don't want to standardize it though

Reply:
great catch! Fixed. Thanks!